package.json

```diff
@@ -96,7 +96,7 @@
     "build-docker-express": "yarn update-dockerfile && docker build --platform=linux/amd64 -t bitgosdk/express:latest -t bitgosdk/express:$(jq -r .version < modules/express/package.json) .",
     "push-docker-express": "docker push bitgosdk/express:latest bitgosdk/express:$(jq -r .version < modules/express/package.json)",
     "update-dockerfile": "ts-node scripts/update-dockerfile.ts",
-    "precommit": "lint-staged"
-  },
-  "dependencies": {}
+    "precommit": "lint-staged",
+    "fix-lint": "lerna run lint -- --fix"
```

Comment:
The `--parallel` or `--stream` option[^1] may be helpful here, similar to the `lint` script. Also it'd be `lint-fix` per convention, similar to `lint-changed`

[^1]: https://github.com/lerna/lerna/blob/main/commands/run/README.md#--stream